src/app/components/ScrollablePromo/Promo/index.jsx

```diff
@@ -70,22 +78,25 @@ const OperaPromoBox = styled.div`
   }
 `;
 
-const Promo = ({ block, onClick }) => {
+const Promo = ({ block, onClick, isRecommendationType }) => {
   const { script, service } = useContext(ServiceContext);
-  const textBlock = filterForBlockType(
-    pathOr({}, ['model', 'blocks'], block),
-    'text',
-  );
-  const href = pathOr(
-    '',
-    ['model', 'blocks', '0', 'model', 'blocks', '0', 'model', 'locator'],
-    textBlock,
-  );
-  const title = pathOr(
-    '',
-    ['model', 'blocks', '0', 'model', 'blocks', '0', 'model', 'text'],
-    textBlock,
-  );
+  const textBlock = isRecommendationType
+    ? block
+    : filterForBlockType(pathOr({}, ['model', 'blocks'], block), 'text');
+  const href = isRecommendationType
+    ? pathOr('', ['locators', 'assetUri'], textBlock)
+    : pathOr(
+        '',
+        ['model', 'blocks', '0', 'model', 'blocks', '0', 'model', 'locator'],
+        textBlock,
+      );
+  const title = isRecommendationType
+    ? pathOr('', ['headlines', 'headline'], textBlock)
+    : pathOr(
+        '',
```

Comment:
If there isn't a fallback value, probably better to use `path` instead

Reply:
fixed